Toggl.Foundation.MvvmCross/ViewModels/TimeEntriesLog/DaySummaryViewModel.cs

```diff
@@ -19,5 +19,22 @@ public DaySummaryViewModel(DateTime day, string title, string totalTrackedTime)
             TotalTrackedTime = totalTrackedTime;
             Identity = new DayHeaderKey(day);
         }
+
+        public bool Equals(DaySummaryViewModel other)
+        {
+            if (ReferenceEquals(null, other)) return false;
+            if (ReferenceEquals(this, other)) return true;
+            return string.Equals(Title, other.Title) && string.Equals(TotalTrackedTime, other.TotalTrackedTime) && Equals(Identity, other.Identity);
+        }
+
+        public override bool Equals(object obj)
```

Comment:
This can be simplified to:

```suggestion
        public override bool Equals(object obj) => (obj as DaySummaryViewModel);
```

Reply:
yeah, I can do that. I used the code generated by Rider because of laziness :)